Makefile

```diff
@@ -142,8 +142,8 @@ export GMON GMONLDOPT
 endif
 
 AFLAGS			+= -D__ASSEMBLY__
-CFLAGS			+= $(USERCFLAGS) $(WARNINGS) $(DEFINES) -iquote include/
-HOSTCFLAGS		+= $(WARNINGS) $(DEFINES) -iquote include/
+CFLAGS			+= $(USERCFLAGS) $(WARNINGS) $(DEFINES) -iquote include/ -std=gnu11
+HOSTCFLAGS		+= $(WARNINGS) $(DEFINES) -iquote include/ -std=gnu11
```

Comment:
@adrianreber should we update `Makefile.config` instead?

Reply:
Not sure. It is not about configuration, but I just saw that it breaks CentOS 7 builds completely. So maybe we cannot use it. Yet.